docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3'
+
+services:
+  web:
+    environment:
+        - DJANGO_DEV=1
+        - VC_DB_HOST=db
+    build: .
+    command: bash -c "python manage.py makemigrations && python manage.py migrate && python manage.py runserver 0.0.0.0:8000"
```

Comment:
Why the "web" service have to "makemigrations"?

Reply:
I'm not sure what alternative you are suggesting.

 `web` service has only the django code(not the db) hence I don't see any other way we can do migrations.